deepspeed/launcher/multinode_runner.py

```diff
@@ -74,7 +74,7 @@ def name(self):
     def get_cmd(self, environment, active_resources):
         environment['PDSH_RCMD_TYPE'] = 'ssh'
         if self.args.ssh_port is not None:  # only specify ssh port if it is specified
-            environment["PDSH_SSH_ARGS_APPEND"] += f" -p {self.args.ssh_port}"
+            environment["PDSH_SSH_ARGS_APPEND"] = f"{environment.get('PDSH_SSH_ARGS_APPEND', '')} -p {self.args.ssh_port}"
```

Comment:
@Lzhang-hub - you'll need to run the pre-commit hooks/formatter on this code to get it to pass the CI checks

Reply:
ok,it passed